model/host/host.go

```diff
@@ -921,3 +945,22 @@ func FindHostsSpawnedByBuild(buildID string) ([]Host, error) {
 	}
 	return hosts, nil
 }
+
+// ComputeParentsToDecommission calculates how many excess parents to decommission
+func ComputeParentsToDecommission(nParents, nContainers, nContainersPerParent int) (int, error) {
```

Comment:
I worry about bugs caused by 3 int arguments.

does it make sense to make this a method on the distro object?